README.md

```diff
@@ -19,7 +19,7 @@ defmodule Posts do
   use Ecto.Filters, key: :search
```

Comment:
It might make sense for the main usage example to call out in a comment that you're overriding the default `filters` key, or maybe just use the default here and cover the override elsewhere. (In general, it seems like this is a case where the extra configurability isn't really necessary for most people esp if the default is the most appropriate name we can think of.)